### DISCUSSION TITLE
Geo-blocked

### DISCUSSION THREAD

**Discussion by azapata13** [iptv-org/community#554] [Q&A]
Is it possible to have access to geo-blocked channels? Otherwise a VPN.

**Comment by BellezaEmporium** [comment]
If you point a VPN or a proxy to the streaming location, yes. Otherwise, no. Unless you manage to find a "questionably legitimate" source that streams it while the original stream is blocked.

**Reply by trivjedn00m** [reply]
VPN helped me.

**Reply by Alpersonne** [reply]
I live in France but i can not get access to geoblocked channels any ideas about how to solve this ?
